public_html/wp-content/mu-plugins/blocks/assets/src/shared/versatile-select/index.js

```diff
@@ -19,6 +19,10 @@ import './style.scss';
 class VersatileSelect extends Component {
 	constructor( props ) {
 		super( props );
+		// do we need to pass props here, or is just calling super() enough? i think it should be if all we want is to access `this`
+		// passing `props` is only needed when we want to reference `this.props` inside the constructor
+		// https://stackoverflow.com/questions/30571875/whats-the-difference-between-super-and-superprops-in-react-when-using-e
+		// if we change it here, we should change the other instances as well
```

Comment:
> do we need to pass props here, or is just calling super() enough?

I don't know. It's worth testing.

Reply:
It worked in my limited testing, but I wanted to check if there was a reason that I was missing. It sounds like it'll be ok to remove it though.